src/pages/Products/Products.js

```diff
@@ -1,10 +1,77 @@
-import { useLayoutEffect } from 'react';
+import { useLayoutEffect, useState } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import GridProducts from 'components/Products/GridProducts';
+import { getProducts } from 'store/products/selectors';
+import SectionHeader from 'components/SectionHeader';
+import ButtonSlim from 'components/UI/ButtonSlim';
+import Wrapper from 'components/UI/Wrapper/Wrapper';
+import ButtonSlimContainer from 'components/UI/ButtonSlimContainer';
 
 const Products = () => {
+  const { productCategory } = useParams();
+  const [isActive, setIsActive] = useState(productCategory);
+  const navigate = useNavigate();
+
+  const products = useSelector(getProducts);
+
   useLayoutEffect(() => {
     window.scrollTo(0, 0);
   }, []);
-  return <div>All Products</div>;
+
+  useLayoutEffect(() => {
+    setIsActive(productCategory);
+  }, [productCategory]);
+
+  const redirectToSeatsCategory = () => {
+    navigate(`/products/seats`);
+  };
+  const redirectToBedsCategory = () => {
+    navigate(`/products/beds`);
+  };
+  const redirectToOfficeCategory = () => {
+    navigate(`/products/office`);
+  };
+  const redirectToKitchenCategory = () => {
+    navigate(`/products/kitchen`);
+  };
+  const redirectToAllCategory = () => {
+    navigate(`/products/all`);
```

Comment:
Let's discuss it.

`const redirect = (category) => navigate(`/products`/${category})`